BSP/STM32F413/Src/BSP_GPIO.c

```diff
@@ -0,0 +1,102 @@
+/* Copyright (c) 2021 UT Longhorn Racing Solar */
+
+#include "BSP_GPIO.h"
+#include "stm32f4xx.h"
+#include "Tasks.h"
+
+OS_SEM GPIO_Update_Sem4;

```

Comment:
Where is this used?

Reply:
Not used, it was used in a previous version of the file, I'll delete it